scripts/add-breached-emails.js

```diff
@@ -16,8 +16,6 @@ if (args["--help"]) {
   console.log("Usage: node add-breached-emails.js [--extraTestEmail=<...>]");
   console.log("Adds test[1-3]@test.com emails to LinkedIn, Adobe, and AllMusic breaches.");
   console.log("--extraTestEmail also adds the supplied test email address and includes it in the LinkedIn, Adobe, and AllMusic breaches.");
-  // We can `process.exit()` here since it's a CLI script.
-  // eslint-disable-next-line no-process-exit
```

Comment:
This rule is already disabled for any scripts/*.js files via the .eslintrc config file. Removing the noise/funk to keep things a bit cleaner.